### PR TITLE
Add dry run option

### LINKED ISSUE #31: Add dry-run option
Add dry-run option: analyse and try to perform obfuscation but do update the images.

### PR DESCRIPTION
**Pull request by kjamroz-bt**
Closes #31
Must be merged after #39

### CODE REVIEW

File: MachObfuscator/Options/Options.swift [hunk 1]
```diff
@@ -140,6 +121,7 @@ extension Options {
           -q, --quiet             quiet mode, no output to stdout
           -v, --verbose           verbose mode, output verbose info to stdout
           -d, --debug             debug mode, output more verbose info to stdout
+          --dry-run               analyze only, do not save obfuscated images
```
Comment (kam800) [inline MachObfuscator/Options/Options.swift]: What do you think about changing `images` -> `files`? `MachObfuscator` also obfuscates NIBs.

File: MachObfuscator/Controller/Obfuscator.swift [hunk 2]
```diff
@@ -67,11 +53,12 @@ class Obfuscator {
             var image: Image = try! loader.load(forURL: obfuscableImage)
             image.replaceSymbols(withMap: manglingMap, paths: paths)
             // TODO: add option
+            // TODO: what is the difference between eraseSymtab and last operation in replaceSymbols?
```
Comment (kam800) [inline MachObfuscator/Controller/Obfuscator.swift]: Nice catch. We probably could remove `symtab` references from `replaceSymbols` a little bit. Alternatively we could add an option for the user to decide about the obfuscation. Do you thing we should create a ticket for that, or should we just leave the TODO here?

File: MachObfuscator/Options/Options.swift (hunk 1 above)
Comment (kjamroz-bt) [reply MachObfuscator/Options/Options.swift]: I agree. Changed.

File: MachObfuscator/Controller/Obfuscator.swift (hunk 2 above)
Comment (kjamroz-bt) [reply MachObfuscator/Controller/Obfuscator.swift]: I implemented option for erasing (actually preserving) symtab, will push it soon.

**Comment by kam800** [comment]
Thanks :)

**Pull request merged**